CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+- Pull Requests should be raised for any change
+- Pull Requests need approval of a [core contributor](https://next-auth.js.org/contributors#core-team) before merging
+- Rebasing in Pull Requests is preferred to keep a clean commit history (see below)
+- Run `npm run lint:fix` before committing to make resolving conflicts easier (VSCode users, check out [this extension](https://marketplace.visualstudio.com/items?itemName=chenxsan.vscode-standardjs) to fix lint issues in development)
+- We encourage you to test your changes, and if you have the opportunity, please make those tests part of the Pull Request
+- If you add new functionality, please provide the corresponding documentation as well and make it part of the Pull Request
+
+### Setting up local environment
+
+A quick guide on how to setup _next-auth_ locally to work on it and test out any changes:
+
+1. Clone the repo:
+
+```sh
+git clone git@github.com:nextauthjs/next-auth.git
+cd next-auth
+```
+
+2. Install packages:
+
+```sh
+npm i
+```
+
+3. Populate `.env.local`:
+
+   Copy `.env.local.example` to `.env.local`, and add your env variables for each provider you want to test.
+
+> NOTE: You can add any environment variables to .env.local that you would like to use in your dev app.
+> You can find the next-auth config under`pages/api/auth/[...nextauth].js`.
+
+1. Start the dev application/server and CSS watching:
+
+```sh
+npm run dev
+```
+
+Your dev application will be available on `http://localhost:3000`
+
+That's it! 🎉
+
+If you need an example project to link to, you can use [next-auth-example](https://github.com/iaincollins/next-auth-example).
+
+#### Hot reloading
+
+When running `npm run dev`, you start a Next.js dev server on `http://localhost:3000`, which includes hot reloading out of the box. Make changes on any of the files in `src` and see the changes immediately.
+
+> NOTE: When working on CSS, you will need to manually refresh the page after changes. (Improving this through a PR is very welcome!)
+
+#### Databases
+
+Included is a Docker Compose file that starts up MySQL, Postgres, and MongoDB databases on localhost.
+
+It will use port `3306`, `5432`, and `27017` on localhost respectively; please make sure those ports are not used by other services on localhost.
+
+You can start them with `npm run db:start` and stop them with `npm run db:stop`.
+
+You will need Docker and Docker Compose installed to be able to start / stop the databases.
+
+When stopping the databases, it will reset their contents.
+
+#### Testing
+
+Tests can be run with `npm run test`.
+
+Automated tests are currently crude and limited in functionality, but improvements are in development.
+
+Currently, to run tests you need to first have started local test databases (e.g. using `npm run db:start`).
+
+The databases can take a few seconds to start up, so you might need to give it a minute before running the tests.
```

Comment:
Wouldn't this be different for the adapters since using `lerna`?

Reply:
Yep I just ripped `CONTRIBUTING.md` and `CODE_OF_CONDUCT.md` from the main repo